lib/rules.js

```diff
@@ -84,7 +84,7 @@ LiveRules.prototype.getAll = function (cb) {
     if (err) cb(err);
     else if (response.statusCode >= 200 && response.statusCode < 300) {
       try {
-        var rules = self.parser.parse(body).rules;
+        var rules = self.live.parser.parse(body).rules;
```

Comment:
Is this correct? I don't see a `self.live` property.

Reply:
It's referring to the object set on line 132 with the property set on line 10.

Reply:
I don't get it. You are referring to the `LiveRules` object, which does not have a `live` property. It would refer to itself...
Did you mean `self.options.parser`?
Not sure how it has been working all this time then 😕

Reply:
Oh yes you are correct, I didn't notice when I made the changes that this line referred to the LiveRules object rather than GnipRules.

The functions within GnipRules have not been working which was why I requested these changes.